forc-pkg/src/manifest/build_profile.rs

```diff
@@ -57,6 +59,7 @@ impl BuildProfile {
             profile: false,
             metrics_outfile: None,
             include_tests: false,
+            log_generation: false,
```

Comment:
```suggestion
            log_generation: true,
```

Reply:
https://github.com/FuelLabs/sway/pull/6996/commits/18d49489bb6899d3fbb19b361d546a01d4ffe3fe

Reply:
We might want to update PR description:

| forc build will still continue to fail for predicates which contain logs (current behaviour)

I think we build in debug mode (not exactly sure) by default so it will build